ExtraDry/ExtraDry.Blazor/Components/Standard/TriSwitch.razor.css

```diff
@@ -0,0 +1,34 @@
+﻿div label {
+  display: flex;
+  align-items: center;
+}
+div label span:first-of-type {
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  width: 40px;
+  height: 20px;
```

Comment:
This feels a bit small for a default.  I think the default should be touch friendly, which is typically 40px x 40px.  Maybe this isn't the whole touch target in which case nevermind...

Reply:
The change event will occur if the touch is either on the swtich or the label.  I think that should be good as a touch target.